vpr/src/route/route_timing.cpp

```diff
@@ -1007,7 +1007,12 @@ bool timing_driven_route_net(ClusterNetId net_id,
     for (int ipin : remaining_targets) {
         if (timing_info) {
             auto clb_pin = cluster_ctx.clb_nlist.net_pin(net_id, ipin);
-            pin_criticality[ipin] = calculate_clb_net_pin_criticality(*timing_info, netlist_pin_lookup, clb_pin);
+            if (!route_ctx.is_clock_net[net_id]) {
+                pin_criticality[ipin] = calculate_clb_net_pin_criticality(*timing_info, netlist_pin_lookup, clb_pin);
```

Comment:
Add comment on why you're using max_criticality (don't get reasonable criticalities for clocks; really want low skew which is approximated by routing them for min delay, particularly if they are on a dedicated network).

Reply:
Add comment